app/controllers/editorial/news_controller.rb

```diff
@@ -6,7 +6,7 @@ class NewsController < EditorialController
 
     def index
       # TODO: return news editorial index
-      bustable_fresh_when(@sections)
+      with_caching(@sections)
```

Comment:
No, but from reading the code it does't render any news articles on the editorial news index so it shouldn't matter.